closed/src/java.base/share/classes/openj9/internal/criu/NativePRNG.java

```diff
@@ -253,116 +245,51 @@ private byte[] implGenerateSeed(int numBytes) {
 
         // supply random bytes to the OS
         // write to "seed" if possible
-        private void implSetSeed(byte[] seed) {
-            synchronized (LOCK_SET_SEED) {
-                if (seedOutInitialized == false) {
-                    seedOutInitialized = true;
-                    seedOut = AccessController.doPrivileged(
-                            new PrivilegedAction<>() {
-                        @Override
-                        public OutputStream run() {
-                            try {
-                                return new FileOutputStream(seedFile, true);
-                            } catch (Exception e) {
-                                return null;
-                            }
-                        }
-                    });
-                }
-                if (seedOut != null) {
-                    try {
-                        seedOut.write(seed);
-                    } catch (IOException e) {
-                        // ignored, on Mac OS X, /dev/urandom can be opened
-                        // for write, but actual write is not permitted
-                    }
-                }
-            }
+        // always add the seed to our mixing random
+        synchronized private void implSetSeed(byte[] seed) {
```

Comment:
Please list modifiers in canonical order (`private synchronized` here; see `java.lang.reflect.Modifier` for the full order).